src/common/utils/utils.ts

```diff
@@ -33,6 +35,27 @@ function getLastMidnightTimestamps(daysBack: number, startFromTonight = false):
     .reverse();
 }
 
+const convertMonetaryAmountToUsdBig = <T extends CurrencyExt>(
```

Comment:
For now this is okay, but we need to revise our utils and build a more compact version of them.

Reply:
Yes for sure, we can refactor it later :+1: